tests/contrib/elasticsearch/test_elasticsearch.py

```diff
@@ -242,6 +242,22 @@ def test_user_specified_service(self):
         assert len(spans) == 1
         assert spans[0].service != "mysvc"
 
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_SERVICE_MAPPING="elasticsearch:custom-elasticsearch"))
+    def test_service_mapping_config(self):
+        """
+        When a user specifies a service mapping it should override the default
+        """
+        from ddtrace import config
+
+        assert config.service != "custom-elasticsearch"
```

Comment:
```suggestion
        assert config.elasticsearch.service != "custom-elasticsearch"
```

also we should just get rid of this assertion because it's not checking anything relevant to the integration.